### ISSUE TITLE
Focus indicator is not visible on the table when focus lands on the table while navigating using tab key.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7231]
Issue:
Focus indicator is not visible on the table when focus lands on the table while navigating using tab key.

Test Environment:
OS: Windows_11
Version: 21H2
OS Build: 22000.795
Browser: Version 103.0.1264.71 (Official Build) (64-bit)

Repro steps:
1. Open https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering in edge browser.
2. Angular excel style filtering page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that the keyboard focus is visible on the complete table content.

Actual Result:
The focus indicator is not visible on the table when the focus lands on the table while navigating using the tab key.

Expected Result:
<img width="957" alt="MAS2 4 7_focus not visible on table (1)" src="https://user-images.githubusercontent.com/93735775/182613124-e5c9f784-fe37-4455-bc61-2b3732f4e8c6.png">

The focus indicator should be visible on the table when the focus lands on the table while navigating using the tab key.

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
